libs/explorers/ui/src/lib/components/section/section.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="section">
```

Comment:
as a suggestion, we can use <section> in places such as this to give this semantic meaning.

Reply:
@sagely1 "<>" should be surrounded by quotes. Classic mistake :-D